spring-cloud-starter-stream-sink-jdbc/src/main/java/org/springframework/cloud/stream/app/jdbc/sink/JdbcSinkConfiguration.java

```diff
@@ -83,7 +103,29 @@
 	private EvaluationContext evaluationContext;
 
 	@Bean
+	public MessageChannel toSink() {
+		return new DirectChannel();
+	}
+
+	@Bean
+	@Primary
 	@ServiceActivator(autoStartup = "true", inputChannel = Sink.INPUT)
+	FactoryBean<MessageHandler> aggregatorFactoryBean(MessageChannel toSink, MessageGroupStore messageGroupStore) {
+		AggregatorFactoryBean aggregatorFactoryBean = new AggregatorFactoryBean();
+		aggregatorFactoryBean.setCorrelationStrategy(
+				new ExpressionEvaluatingCorrelationStrategy("payload.getClass().name"));
+		aggregatorFactoryBean.setReleaseStrategy(new MessageCountReleaseStrategy(properties.getBatchSize()));
+    aggregatorFactoryBean.setGroupTimeoutExpression(new ValueExpression<>(properties.getIdleTimeout()));
```

Comment:
Wrong indent.

Reply:
Should be fixed in 8d33bd5c35048c2385412e905ebfd96ff43c3b16 .